src/Introspection/Utilities.hack

```diff
@@ -0,0 +1,151 @@
+namespace Slack\GraphQL\Introspection;
+
+final abstract class Utilities {
+    const type TIntrospectionOptions = shape(
+        /**
+         * Whether to include descriptions in the introspection result.
+         * Default: true
+         */
+        ?'descriptions' => bool,
+
+        /**
+         * Whether to include `specifiedByURL` in the introspection result.
+         * Default: false
+         */
+        ?'specified_by_url' => bool,
+
+        /**
+         * Whether to include `isRepeatable` flag on directives.
+         * Default: false
+         */
+        ?'directive_is_repeatable' => bool,
+
+        /**
+         * Whether to include `description` field on schema.
+         * Default: false
+         */
+        ?'schema_description' => bool,
+
+        /**
+         * Whether target GraphQL server support deprecation of input values.
+         * Default: false
+         */
+        ?'input_value_deprecation' => bool,
+    );
+
+    public static function getIntrospectionQuery(self::TIntrospectionOptions $options = shape()): string {
+        $descriptions = ($options['descriptions'] ?? false) ? 'description' : '';
```

Comment:
These would be nicer as actual GraphQL variables in the query, e.g.

```GraphQL
query IntrospectionQuery($include_descriptions: Boolean = true) {
  # ...
  description @include(if: $include_descriptions)
```

but we haven't implemented `@inculde` and `@skip` yet so we may have to address this later.